packages/contracts/src/strategies/BaseStrategy.sol

```diff
@@ -158,13 +158,15 @@ abstract contract BaseStrategy is
         _adjustPosition(outstandingDebt);
 
         uint256 totalDebt = lender.currentDebt();
+        uint256 gasPrice = _gasPriceUSD() * estimatedWorkGas;
```

Comment:
Can you please make this as separate function? This logic was used a few times in separate places. And maybe more correct to name it as gasCost. 
This is minor, just suggestion